README.md

```diff
@@ -23,24 +23,18 @@ It is possible to call `setFocusBehavior()` arbitrarily many times before callin
 ```js
 // Example #1 - unconditional decision.
 
-const controller = new CaptureController;
+const controller = new CaptureController();
```

Comment:
Thanks for catching.

Reply:
👍

Reply:
Note that it works though ;)

Reply:
Wait, so...?

Reply:
It is not common to see that. Let's use `new CaptureController();`